### ISSUE TITLE
Quickstart: Deploy Hello World not working

### ISSUE THREAD

**Issue by TobiasNissen** [Azure/service-fabric-mesh-preview#391]
I've been following the quickstart guide at [https://docs.microsoft.com/en-us/azure/service-fabric-mesh/service-fabric-mesh-quickstart-deploy-container](https://docs.microsoft.com/en-us/azure/service-fabric-mesh/service-fabric-mesh-quickstart-deploy-container).

When reaching the "Deploy the Application" section, I run into trouble. In particular, the command: 
`az mesh deployment create --resource-group myResourceGroup --template-uri https://raw.githubusercontent.com/Azure-Samples/service-fabric-mesh/master/templates/helloworld/helloworld.linux.json --parameters "{'location': {'value': 'eastus'}}" 
` 
terminates with the following error:
`ValidationError: The request content was invalid and could not be deserialized: 'Could not find member 'template' on object of type 'DeploymentDefinition'. Path 'template', line 1, position 12.`

I have of course ensured that a valid resource group is referenced. I haven't been able to find any information on the missing "template" member.

**Comment by LokeshSoundarajan** [comment]
I'm also facing the same issue at the deploying application step. I have tried with both linux and windows templates. But its the same error.
ValidationError: The request content was invalid and could not be deserialized: 'Could not find member 'template' on object of type 'DeploymentDefinition'. Path 'template', line 1, position 12.

**Comment by cankanat** [comment]
I am having the same problem not only in hello world template but also in other templates.

"BadRequestError: The request content was invalid and could not be deserialized: 'Could not find member 'template' on object of type 'DeploymentDefinition'. Path 'template', line 1, position 12.'."

**Comment by CloudDevDan** [comment]
I've also experienced this issue today.

**Comment by aleks-noname** [comment]
Yes, I'm also trying deploy all samples and got same error. Anybody can resolve it?

**Comment by cankanat** [comment]
I solve this issue with using **Deploy Custom Template**. You can paste the template file to the template editor.  I don't know exactly why, but when I did it this way, the deployment was successfully done.


![image](https://user-images.githubusercontent.com/17693535/101986839-78432680-3ca1-11eb-9898-2587916caf32.png)

**Comment by aleks-noname** [comment]
Hi @cankanat.
Thank you for answer!
Yes, It's help me. The deployment started but I got another error during deployment application (gateway and network is ok):
{
    "status": "Failed",
    "error": {
        "code": "ClusterAllocationInsufficientCapacity",
        "message": "Insufficient capacity is available in this region. Please try again later or try a different region.",
        "details": []
    }
}

I'm trying two region West Europe and East US, checked quotas and nothing.
Could you know something about it?

**Comment by aleks-noname** [comment]
Hi @cankanat.
Thank you for answer!
Yes, It's help me. The deployment started but I got another error during deployment application (gateway and network is ok):
{
    "status": "Failed",
    "error": {
        "code": "ClusterAllocationInsufficientCapacity",
        "message": "Insufficient capacity is available in this region. Please try again later or try a different region.",
        "details": []
    }
}

I'm trying two region West Europe and East US, checked quotas and nothing.
Could you know something about it?

**Comment by aleks-noname** [comment]
Hi.
I got answer from MS Support:
_Service Fabric Mesh preview is temporarily disabled due to security reasons.
While the service is disabled, if you attempt to utilize the Mesh service you will receive the following error message: "Cannot deploy Service Fabric Mesh application in region, reports insufficient capacity"._

Waiting when Microsoft guys will enable SFM.
